pkg/connectors/openapiclient/api/openapi.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: 3.0.1
+info:
+  license:
+    name: Apache 2.0
+  title: Policy Manager Service
```

Comment:
same comment as above. not sure if this resolved openapi.yaml file is required. @roee88 can you please give your opinion on this? Thanks.

Reply:
Currently `api` folder only contains `openapi.yaml` file. So, deleted the entire `api` folder. Is that fine ? @rohithdv